custom_components/meteoswiss/weather.py

```diff
@@ -68,7 +64,7 @@ def __init__(
     def __set_data(self, data: MeteoSwissClientResult) -> None:
         self._displayName = data[CONF_FORECAST_NAME]
         self._forecastData = data["forecast"]
-        self._condition = data["condition"]
+        self._condition = data["condition"][0]
```

Comment:
This sometimes fails.

I've implemented a fix for this in master.  Please rebase onto master and I will merge this.

Reply:
Done.

Reply:
Shall I call `condition_name_to_value` on `data["condition"]` or is it fine now?